analytics/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
There is an `.gitignore` file in the root directory, so maybe we can remove this.